iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/DataNode.java

```diff
@@ -449,7 +446,7 @@ private void storeRuntimeConfigurations(
     initTTLInformation(runtimeConfiguration.getAllTTLInformation());
 
     /* Store cluster ID */
-    IoTDBDescriptor.getInstance().getConfig().setClusterId(runtimeConfiguration.getClusterId());
+    storeClusterID(runtimeConfiguration.getClusterId());
```

Comment:
After remove the cluster id from DN restart resp, NPE might happen here

Reply:
fixed.